basis-universal/src/transcoding/enums.rs

```diff
@@ -289,17 +292,32 @@ impl TranscoderTextureFormat {
         output_row_pitch_in_blocks_or_pixels: Option<u32>,
         output_rows_in_pixels: Option<u32>,
     ) -> bool {
-        unsafe {
-            sys::basis_validate_output_buffer_size(
-                self.into(),
-                output_blocks_buf_size_in_blocks_or_pixels,
-                original_width,
-                original_height,
-                output_row_pitch_in_blocks_or_pixels.unwrap_or(0),
-                output_rows_in_pixels.unwrap_or(0),
-                total_slice_blocks,
-            )
+        if !self.is_compressed() {
```

Comment:
I'd rather we call the C implementation that try to duplicate it for same reasons as in previous comment